metrics/metrics.go

```diff
@@ -19,6 +19,7 @@ const (
 	UnkownKind Kind = iota
 	CodaHaleKind
 	PrometheusKind
+	AllKind
```

Comment:
it's good like this, but possible cosmetics change:

```
UnknownKind Kind = 0
CodaHaleKind Kind = 1 << iota
PrometheusKind
AllKind = CodaHaleKind | PrometheusKind
```

this way it is easier to add future kinds.